docs/whats-new.asciidoc

```diff
@@ -13,75 +13,71 @@ Other versions: {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-a
 [float]
 == Detection rules enhancements
 
-*Alert suppression for custom query rules*
+*Alert suppression time window for custom query rules*
 
-{security-guide}/alert-suppression.html[Alert suppression] allows you to reduce the number of repeated or duplicate detection alerts created by a custom query detection rule each time the rule runs. Matching events are grouped by a specified field, and only one alert is created for each group per rule execution. You can also specify multiple fields to group events by unique combinations of values. 
+You can now {security-guide}/alert-suppression.html[suppress] custom query rule alerts during a specific time window to limit alert volume and further reduce alert noise. Duplicate alerts within that time window will be grouped.
 
-*New terms rules allow multiple fields*
+*Connectors can be exported and imported with rules*
 
-You can now configure {security-guide}/rules-ui-create.html#create-new-terms-rule[new terms] detection rules to detect combinations of up to three new terms. For example, a host IP address and host ID that haven't been observed together. 
+Now, when you {security-guide}/rules-ui-management.html#import-export-rules-ui[export and import] a rule that uses action connectors you don't need to handle the connectors separately. Note that sensitive information about the connector (such as authentication credentials) is not included in the export file. You must re-add missing connector details after importing detection rules.
 
-*New Tines connector for rule actions* 
+*Cross-cluster search is supported for indicator match rules*
 
-There's a new Tines connector you can select when configuring rule actions. This connector uses Tines' https://www.tines.com/docs/actions/types/webhook[Webhook Actions] to send events to a Tines story.
+You can now query data in remote clusters and cold-tier data for index patterns specified in indicator match rules.
 
-*{ml-cap} job enhancements*
+*Enhancements to rule preview*
+
+The {security-guide}/rules-ui-create.html#preview-rules[Rule preview] panel can be resized.
 
-The following enhancements simplify managing {ml} detection rules and their associated {ml} jobs:
+*New Torq connector for rule actions* 
```

Comment:
@jmikell821 I'm not sure whether we should include this. When I talked to @lcawl ~3 weeks ago about documenting this in our release docs (comment [here](https://github.com/elastic/security-docs/pull/3049#discussion_r1131714010)), we agreed it was prob best to only mention the connector in the release notes. But then this morning, I noticed that this new connector was being highlighted as a new feature in the 8.7 Security TOI. How should we handle documenting this new connector?

Reply:
@nastasha-solomon I think it's fine to just mention this in release notes. Thanks for checking!